Ports/alpine/patches/0002-Be-robust-to-no-aliases.patch

```diff
@@ -0,0 +1,22 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Taj Morton <tajmorton@gmail.com>
+Date: Sun, 27 Nov 2022 17:30:49 -0800
+Subject: [PATCH] Be robust to no aliases.
+
+---
+ pith/osdep/domnames.c | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/pith/osdep/domnames.c b/pith/osdep/domnames.c
+index c7ea177e65ae08d6163c962934973e966dd54c83..536fa8f3d8dae5d61c306afe49c32233ddc9ddce 100644
+--- a/pith/osdep/domnames.c
++++ b/pith/osdep/domnames.c
+@@ -85,7 +85,7 @@ getdomainnames(char *hostname, int hsize, char *domainname, int dsize)
+ 	 */
+ 	strncpy(hostname, he->h_name, hsize-1);
+ 	hostname[hsize-1] = '\0';
+-	if(strchr(hostname, '.') == NULL){		/* no dot in hostname */
++	if(strchr(hostname, '.') == NULL && he->h_aliases != NULL){		/* no dot in hostname */
```

Comment:
Why is this patched if you also make a LibC change for it?

Reply:
Just an artifact of the evolution of this branch, and not being sure if patching libc was the right way to go. This patch has now been dropped. I've built and re-tested.